test/helpers.js

```diff
@@ -67,34 +110,87 @@ export const stripPositions = function(expr) {
     return expr;
 };
 
-export const parseAndSetResult = function(expr, result,
-                                          settings = new Settings()) {
+export const Mode = {
+    PARSE: {
+        apply: parseTree,
+        get: getParsed,
+        noun: 'parsing',
+        Verb: 'Parse',
+    },
+    BUILD: {
+        apply: _getBuilt,
+        get: getBuilt,
+        noun: 'building',
+        Verb: 'Build',
+    },
+};
+
+const getTree = (expr, settings, mode) => {
+    const result = expectKaTeX(expr, settings, mode);
     try {
-        return parseTree(expr, settings);
+        expect(result).toHavePassed();
```

Comment:
I would expect `getTree` to call `katex.render()` and return the tree or clean up the stack trace and rethrow as necessary.  Then `expectKaTeX` can call `getTree` and we don't have introduce a special matcher just to make things work in this order.  Is there something I'm missing that necessitates this setup?